pluto/docker-compose.yml

```diff
@@ -0,0 +1,95 @@
+services:
+  app_proxy:
+    environment:
+      APP_HOST: pluto_frontend_1
+      APP_PORT: 7777
+    depends_on:
+      - frontend
+    restart: on-failure
+
+  discovery:
+    image: registry.gitlab.com/plutomining/pluto/pluto-discovery:0.6.0-rc.1@sha256:3da84654b98c07a64ffc93f14c525a742753f6afe776526de3adcf9c728169d0
+    network_mode: "host"
+    pid: host
```

Comment:
Do you know if `pid` is required in all the containers, or can we remove it? The goal here is to keep container permissions as low as possible without impacting app functionality